vllm/model_executor/models/qwen.py

```diff
@@ -8,7 +8,7 @@
 The input of the model is flattened to a 1D tensor of tokens. The model uses
 InputMetadata to extract the original 2D shape of the input.
 """
-from typing import List, Optional, Tuple
+from typing import List, Optional, Tuple, Dict, Any
```

Comment:
Could you sort the order of the types?
```suggestion
from typing import Any, Dict, List, Optional, Tuple
```

Reply:
sure, fixed